README.md

```diff
@@ -1,13 +1,14 @@
 # rb
 
-With 9 lines of Ruby replace most of the command line tools that you use to process text inside of the terminal.
+With 10 lines of Ruby replace most of the command line tools that you use to process text inside of the terminal.
 
 
 
 Here's the code
 
 ```ruby
 #!/usr/bin/env ruby
+load Dir.home + '/.rbrc' if File.exists?(Dir.home + '/.rbrc')
```

Comment:
Small suggestion: would you consider changing this to...

```ruby
load rbrc if File.exists? rbrc = File.join(Dir.home, '.rbrc')
```

... to keep it a bit more DRY, and a tad more idiomatic as well?

Reply:
In the first part of one-line version of `if` you can't use variables declared in the second one.
You're right about the `File.join` though. This could work:

```ruby
File.join(Dir.home, '.rbrc').tap { |f| load f if File.exists?(f) }
```

Reply:
> In the first part of one-line version of if you can't use variables declared in the second one.

Reply:
Or if you want to go all out...

```ruby
begin load(File.join(Dir.home, '.rbrc')) ; rescue LoadError ; end
```